java/server/src/org/openqa/selenium/grid/node/local/LocalNode.java

```diff
@@ -181,11 +189,53 @@ public boolean isReady() {
   }
 
   @VisibleForTesting
+  @ManagedAttribute(name = "CurrentSessions")
   public int getCurrentSessionCount() {
     // It seems wildly unlikely we'll overflow an int
     return Math.toIntExact(currentSessions.size());
   }
 
+  @ManagedAttribute(name = "MaxSessions")
+  public int getMaxSessionCount() {
+    return maxSessionCount;
+  }
+
+  @ManagedAttribute(name = "Status")
+  public Availability getAvailability() {
+    return isDraining() ? DRAINING : UP;
+  }
+
+  @ManagedAttribute(name = "TotalSlots")
+  public int getTotalSlots() {
+    return factories.size();
+  }
+
+  @ManagedAttribute(name = "UsedSlots")
+  public long getUsedSlots() {
+    return factories.stream().filter(sessionSlot -> !sessionSlot.isAvailable()).count();
+  }
+
+  @ManagedAttribute(name = "Load")
+  public float getLoad() {
+    long inUse = factories.stream().filter(sessionSlot -> !sessionSlot.isAvailable()).count();
+    return inUse / (float) maxSessionCount * 100f;
+  }
+
+  @ManagedAttribute(name = "RemoteNodeUri")
+  public URI getExternalURI() {
```

Comment:
Nit: `getExternalUri` We try and AVOID SHOUTING in Selenium (which is why we have `HttpMethod`, and so on)